packages/html/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I like the idea! Let's land https://github.com/PrairieLearn/PrairieLearn/pull/5123 first, since that lays a small amount of groundwork for using multiple licenses throughout different parts of the codebase. We can come back to this in a small followup PR.

Reply:
Sounds good!